vllm/spec_decode/interfaces.py

```diff
@@ -54,6 +54,10 @@ class SpeculativeScores:
     # Optional last hidden states from the scoring model.
     hidden_states: Optional[torch.Tensor] = None
 
+    # Scoring model may also return logprobs for prompt tokens
+    # for each request, when chunked prefill is enabled.
```

Comment:
Yes previously available features shouldn't be affected by this change. We were able to get away with fewer loc  because we either had all prompts (so no spec, here https://github.com/vllm-project/vllm/blob/main/vllm/spec_decode/spec_decode_worker.py#L654) or all decodes.

Reply:
In particular, addition here was necessary because the Speculator now runs prefills too, so it needed a way to report the prompt_logprobs back to the worker.